resources/wikia/libraries/jquery/autocomplete/jquery.autocomplete.js

```diff
@@ -293,19 +305,28 @@
         return;
       }
 
-      var me, len, div, f, suggestion;
+      var me, len, div, f, suggestion, v, suggestionWrapperElement;
       me = this;
       len = this.options.maxSuggestions && this.options.maxSuggestions < this.suggestions.length ? this.options.maxSuggestions : this.suggestions.length;
       f = this.options.fnFormatResult;
       v = this.getQuery(this.currentValue);
+      // wikia change - start
+      suggestionWrapperElement = this.options.suggestionWrapperElement;
+      // wikia change - end
       this.container.hide().empty();
 
       for (var i = 0; i < len; i++) {
-          // wikia change - start
-          suggestion = this.suggestions[i];
-        div = $((me.selectedIndex === i ? '<div class="' + this.options.selectedClass + '"' : '<div')
-            + ' title="' + $.htmlentities(suggestion) + '">' + f($.htmlentities(suggestion), this.data[i], $.htmlentities(v))
-            + '</div>');
+        // wikia change - start
+        suggestion = this.suggestions[i];
+        div = $(
+          (
+            me.selectedIndex === i ?
+            '<' + suggestionWrapperElement + ' class="' + this.options.selectedClass + '"' :
+            '<' + suggestionWrapperElement
+          )
+          + ' title="' + $.htmlentities(suggestion) + '">' + f($.htmlentities(suggestion), this.data[i], $.htmlentities(v))
+          + '</' + suggestionWrapperElement + '>'
+        );
         // wikia change - end
         div.mouseover((function(xi) { return function() { me.activate(xi); }; })(i));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define this function outside of a loop. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionDefinitionInsideLoop)